dind/docker-compose.yml

```diff
@@ -0,0 +1,93 @@
+version: '3'
+
+services:
+
+    node1:
+        build:
+            context: .
+            args:
+                # from env
+                - DOCKER_GROUP_ID
+        ports:
+            #- "5001:2375"
+            - "8501:8500"
```

Comment:
Exposing consul on 1 node should be enough! 

At least one node should expose ``8500`` port or you will have to change how tests are communicate with consul [while instantiate self.consul in cluster.py](https://github.com/mlfmonde/cluster/pull/43/files#diff-582238d97b7d8ecf826239860fce307fR30)

Reply:
My comments was over since we've decided to run test in a docker container as well and not on the host.